apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -2715,6 +2715,13 @@ export const extensions: Extensions = {
                                             " alphanumeric characters</2><3>begin with an alphabetic character</3>" +
                                             "</3s>"
                                     }
+                                },
+                                adu: {
```

Comment:
Shall we add this to `modules/i18n/src/translations/en-US/portals/tenants.ts`. we are in the process of moving the console extension i18n to i18n module.